### ISSUE TITLE
Use createStoreWithVersion helper

### ISSUE THREAD

**Issue by kumar303** [mozilla/addons-code-manager#763]
See https://github.com/mozilla/addons-code-manager/pull/753/files#r284634744 for a new helper, `createStoreWithVersion()`.

We can use this in a lot of different places. Let's follow-up and add it everywhere. You could search `loadVersionInfo` to find all the tests and stories that would benefit from it.

**Comment by ioanarusiczki** [comment]
@kumar303 @willdurand 
Could you add some STR for qa? 

I've noticed that the browse and compare pages are quite unresponsive while navigating through files today as I mentioned [here](https://github.com/mozilla/addons-code-manager/issues/825#issuecomment-505855445)

**Comment by kumar303** [comment]
@ioanarusiczki I updated it with some QA tips and replied to your questions on https://github.com/mozilla/addons-code-manager/issues/825#issuecomment-505855445 as well

**Comment by ioanarusiczki** [comment]
@kumar303 
Verified this on -dev and -stage with FF67(Win10).
- Browse page looks good to me. 

![Browse page](https://user-images.githubusercontent.com/33448286/60251759-d45d9380-98d1-11e9-8b5b-db559fe336cb.png)

- Compare page on -dev shows the Information card but all the pages I've checked on -stage do not have it (this would show the info for the newest version).  I filed https://github.com/mozilla/addons-code-manager/issues/888
